cli/flox/src/commands/environment.rs

```diff
@@ -1178,19 +1178,20 @@ impl Install {
                 ),
             )) if pkgdberr.exit_code == 120 => 'error: {
                 let paths = packages.iter().map(|p| p.path.clone()).join(", ");
-                let head = format!("❌  could not install {paths}");
 
                 if packages.len() > 1 {
                     break 'error anyhow!(formatdoc! {"
-                        {head}
+                        ❌  Could not install {paths}.
```

Comment:
blocking: the ticket asks for `Could not find package`

Reply:
The issue was about the single package not found...
but I guess we could say 
```suggestion
                        ❌  Could not find one or more packages of {paths}.
```

Reply:
Oh right I misread that